understanding/accessible-authentication-minimum.html

```diff
@@ -125,203 +122,122 @@ <h1><span class="standalone-resource__type-of-guidance">Understanding
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>意図</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Make logins possible with less mental effort.</dd>
+                  <dt>目標</dt>
+                  <dd>より少ない精神的労力でログインできるようにする。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Don't make people solve, recall, or transcribe something to log in.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>ログインするために人々に何かを解かせたり、思い出させたり、転記させたりしない。</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>Some people with cognitive disabilities cannot solve puzzles, memorize a username
-                     and password, or retype one-time passcodes.
+                  <dt>なぜそれが重要か</dt>
+                  <dd>認知障害のある人々の中には、パズルを解くことも、ユーザ名及びパスワードを記憶することも、ワンタイムパスコードを再入力することもできない人もいる。
                   </dd>
                   
                </dl>
             </section>
             <section id="intent">
-               <h2>Intent</h2>
-               <p>The purpose of this Success Criterion is to ensure there is an accessible, easy-to-use,
-                  and secure method for users to authenticate when logging into an existing account.
-                  As the most prevalent form of authentication, Web sites commonly rely on usernames
-                  and passwords to log in. However, memorizing a username and password places a very
-                  high or impossible burden upon people with certain cognitive disabilities, as do additional
-                  steps often added to authentication processes. For instance, the need to transcribe
-                  a one-time verification code or requiring a puzzle to be solved.
+               <h2>意図</h2>
+               <p>この達成基準の目的は、既存のアカウントにログインするときに、アクセシブルで、使いやすく、安全な方法で利用者が認証するのを保証することである。最も広まっている認証形式として、ウェブサイトは通常、ログインするためにユーザ名及びパスワードに依存する。しかし、ユーザ名及びパスワードを記憶することは、認証プロセスにしばしば追加される追加手順と同様に、特定の認知障害のある人々にとって非常に大きな負担、又はひどく困難な負担となる。例えば、ワンタイム認証コードを転記する必要がある場合、パズルを解く必要がある場合などである。
                </p>
-               <p>While Web sites can use the recognition of objects or of non-text content provided
-                  by the user to meet this Success Criterion, such techniques do not fully support the
-                  cognitive accessibility community and should be avoided if possible. Refer to <a href="accessible-authentication-enhanced">Accessible Authentication (Enhanced)</a> for guidance to be more inclusive and accessible.
+               <p>ウェブサイトは、この達成基準を満たすために利用者が提供するオブジェクト又は非テキストコンテンツの認識を使用できるが、そのような技術は認知アクセシビリティコミュニティを完全にはサポートしておらず、可能な限り避けるべきである。より包括的でアクセシブルにするためのガイダンスについては、<a href="accessible-authentication-enhanced">アクセシブルな認証 (高度)</a> を参照すること。
```

Comment:
達成基準 3.3.8 の例外事項を見ると、「物体の認識」と「個人特有のコンテンツ（i.e. その利用者本人がウェブサイトに提供した非テキストコンテンツ）」という箇条書き項目になっているので、ここの原文の privided by the user は non-text content のみにかかっていて、objects にはかかっていないように思います。以下はいかがでしょうか？

```suggestion
               <p>ウェブサイトは、この達成基準を満たすために、物体又は利用者が提供する非テキストコンテンツの認識を使用できるが、そのような技術は認知アクセシビリティコミュニティを完全にはサポートしておらず、可能な限り避けるべきである。より包括的でアクセシブルにするためのガイダンスについては、<a href="accessible-authentication-enhanced">アクセシブルな認証 (高度)</a> を参照すること。
```